tests/pages/services/ServiceActions-cy.js

```diff
@@ -220,10 +217,8 @@ describe("Service Actions", function() {
         });
         cy
           .get(".confirm-modal .button-collection .button-danger")
-          .as("dangerButton")
-          .click();
```

Comment:
Shouldn't we still test the button is disabled once clicked and then reenabled upon faulty request?